src/docs/installation-manual.md

```diff
@@ -9,10 +9,17 @@ on a fresh installation of `Ubuntu 20.02.2 LTS` to run directly on the system wi
 
 ## Preparation
 
-Make sure your computer is up to date.
+Make sure your computer is up to date and install the required dependencies needed to build and run Photoview.
 
 ```shell
-$ sudo apt update && sudo apt upgrade
+$ sudo add-apt-repository ppa:strukturag/libheif
+$ sudo add-apt-repository ppa:strukturag/libde265
+
+$ sudo apt update && sudo apt -y upgrade
+$ sudo apt -y install git curl libdlib-dev libblas-dev liblapack-dev libjpeg-turbo8-dev gcc gpg libdlib19 ffmpeg exiftool libheif1 \
+  g++-aarch64-linux-gnu libc6-dev-arm64-cross libdlib-dev libblas-dev liblapack-dev libjpeg-dev libheif-dev pkg-config \
+  build-essential software-properties-common libheif1 libde265-0
```

Comment:
It's very nice to properly document the required dependencies as this was clearly missing before.
I think that some of these are unnecessary though, how did you gather the list?

Especially `libc6-dev-arm64-cross` seems unnecessary unless one is cross building for arm processors.
From the readme, these are the required dependencies: `libdlib-dev libblas-dev liblapack-dev libjpeg-turbo8-dev libheif-dev`

Also ffmpeg is not necessary and is explained under optional dependencies. Maybe we should move `exiftool` to the optional dependencies section as well, as Photoview can use an internal exif parser if exiftools wasn't found.

Note, that I use macOS for development so my list of dependencies might be wrong or outdated, please correct me if some packages are in fact needed.

Reply:
I thought that since it was a bit difficult for me to find the right dependencies it should be put correctly in the documentation. I used the Dockerfile and also searched the discussions. I'm happy to leave out the suggested dependencies and test that, I use LXC for virtualization.

Reply:
Yes, it would be great to gather a correct list as there doesn't exist one atm.
The docker dependencies are a bit specific as the Dockerfile is used to cross compile for ARM which is irelevant for this guide.

I think `g++-aarch64-linux-gnu` and `g++-aarch64-linux-gnu libc6-dev-arm64-cross` are unneeded as they relate to cross compilation.

I don't know what `software-properties-common` is or if it's needed?

`gcc` is unneeded as it's a part of `build-essential`
`libheif1` and `libde265-0` are unneeded as it's a part of `libheif-dev`

I don't think `git` and `curl` are needed to build the api.

Reply:
I have updated the instructions to the current and correct status.